core/controllers/profile.py

```diff
@@ -412,6 +412,11 @@ class UserInfoHandler(base.BaseHandler):
     @acl_decorators.open_access
     def get(self):
         """Handles GET requests."""
+        # The following headers are added to prevent caching of this response.
+        self.response.headers[b'Cache-Control'] = (
+            b'no-cache, no-store, must-revalidate')
+        self.response.headers[b'Pragma'] = b'no-cache'
+        self.response.headers[b'Expires'] = b'0'
```

Comment:
According to my reading of https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers these three can be replaced by just `self.response.headers[b'Cache-Control'] = 'no-store'` and maybe even `self.response.cache_control.no_store = True` but that needs to be checked in the browser.

Reply:
Yes, the [SO answer](https://stackoverflow.com/a/2068407) describes this. I've added the `Pragma` and `Expires` for supporting older browsers. The MDN docs also mentions that these additional headers enable backwards compatibility with HTTP/1.0 caches where the Cache-Control header is not yet present.

Reply:
I mean I understand trying to hold backward compatibility, but `Pragma` is required for IE6, I don't believe that Oppia will run there anyway (might be worth a shot to test it). And if you look at the comments in the SO answers there are some mentions of the browsers breaking for these settings (https://stackoverflow.com/questions/49547/how-do-we-control-web-page-caching-across-all-browsers/2068407#comment31775854_2068407)

@seanlip btw do we have any list of platforms, browsers and their versions that we support? It might be worth it to look at Oppia in some older browsers and find the first browser version where everything works and set the line there (this might be especially worth it for Internet Explorer)

Reply:
Okay, I tried IE 11 and it doesn't seem to work there (https://www.browserling.com/browse/win/7/ie/11/https%3A%2F%2Fwww.oppia.org)

Reply:
Great analysis @vojtechjelinek, thanks! I also tested on IE11 and everything looks broken (I'm seeing a blank page). Let's go with your suggestion of "no-store". Thanks!

Reply:
@kevintab95, just to check, do you need me to deploy this?

@vojtechjelinek not really -- we just have a loose standard of Chrome + Firefox currently. I agree with you that defining this more clearly would be better, but I feel it might not be super high priority right now. Would you mind filing an issue so that we can keep track, and do an investigation when the other stuff we need to do in the short term dies down a little bit?

Reply:
@seanlip, yep I think we should test it. Can you please deploy these changes? Thanks!

Reply:
@kevintab95 Could you please check if `self.response.cache_control.no_store = True` also works? I think that is is a little bit clearer.

Reply:
Hi @vojtechjelinek, the reason I set headers this way is because we do this the same way in base.py.

Reply:
We use both in `base.py` (https://github.com/oppia/oppia/blob/develop/core/controllers/base.py#L360) but I'm okay with using either way.

Reply:
Made the change, thanks!